### DISCUSSION TITLE
QST: Geopandas on android

### DISCUSSION THREAD

**Discussion by adamekcerv** [geopandas/geopandas#2730] [Q&A]
Hi,

Is it possible to install geopandas on android (pydroid app). 
Thank you.

**Comment by martinfleis** [comment]
That may not be super straightforward (never tried tbh) but if you manage to install the dependencies (shapely, pandas, fiona - not necessary, and pyproj) then installing geopandas will be easy as it is a pure python. But all those dependencies have some compiled code, so it may be trickier.
